monkestation/code/modules/research/designs/medical_designs.dm

```diff
@@ -19,3 +19,13 @@
 	build_path = /obj/item/organ/cyberimp/arm/lighter
 	category = list("Misc", "Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/lethalsyringe
```

Comment:
If these are going to be usable for more than lethal injection now, they could probably use a rename.
One idea is "Titanium Syringe", just going by the materials used.
I've seen one server call them iron spikes.